capture-sdk/sdk/src/main/res/values/colors.xml

```diff
@@ -52,14 +52,16 @@
     <color name="gc_multi_page_review_add_pages_text">#fff</color>
 
     <color name="gc_analysis_activity_background">@android:color/transparent</color>
-    <color name="gc_analysis_fragment_background">@color/gc_background</color>
```

Comment:
[Request] Analysis fragments should use the theme background color. I informed István to fix the theme background color inconsistency between light and dark modes.